### PR TITLE
fix: set ignoredLayers to 4 and added missing outcommented layers for…

### PR DESCRIPTION
**Pull request by OpenDEM**
… documentation.

Some issues can be closed.

### CODE REVIEW

**Comment by wschildbach** [comment]
Hi @OpenDEM , 

I staged this fix at https://test1-alpha.openseamap.org/ ? . Can you take a look whether it works there?

**Comment by OpenDEM** [comment]
Hi @wschildbach, yes, this looks good! Example harbours and gebco depths:
https://test1-alpha.openseamap.org/?zoom=10&lat=54.15300&lon=11.65400&layers=BFTFFTTFFTF0FFFFFFFFFF

File: index.php [hunk 1]
```diff
@@ -97,14 +97,16 @@
             var layer_tidalscale;                  // 16
             var layer_permalink;                   // 17
             var layer_waterdepth_trackpoints_100m;      // 18
+			 // var layer_elevation_profile_contours;  // 19
```
Comment (wschildbach) [inline index.php:100]: Can you harmonize the indentation in this line?

Comment (wschildbach) [reply index.php:100]: I'll merge once the indentation is in-line.

Comment (OpenDEM) [reply index.php]: @wschildbach Just fixed it.

**Comment by wschildbach** [comment]
See also #162 , #157, #91, #24 -- these might potentially be fixed.

File: index.php (hunk 1 above)
Comment (wschildbach) [reply index.php]: Thanks @OpenDEM , this is now fixed. @Nautic , FYI.

**Comment by OpenDEM** [comment]
#162 is fixed, but not the other issues

**Pull request merged**